controllers/akodeploymentconfig/akodeploymentconfig_controller_avi_phase.go

```diff
@@ -127,6 +128,11 @@ func (r *AKODeploymentConfigReconciler) reconcileNetworkSubnets(
 
 	log.Info("Start reconciling AVI Network Subnets")
 
+	if r.aviClient == nil {
+		log.Info("AVI client not initialized, requeue")
+		return res, nil
```

Comment:
If you want the reconciler requeue, you have to throw error, otherwise, it will think it already reach the status you want, it won't re-reconcile

Reply:
done